src/StakeRegistry.sol

```diff
@@ -203,15 +186,91 @@ contract StakeRegistry is VoteWeigherBase, StakeRegistryStorage {
         }
     }
 
-    /*******************************************************************************
-                    EXTERNAL FUNCTIONS - SERVICE MANAGER OWNER
-    *******************************************************************************/
+    /// @notice Initialize a new quorum and push its first history update
+    function initializeQuorum(
+        uint8 quorumNumber,
+        uint96 minimumStake,
+        StrategyAndWeightingMultiplier[] memory strategyParams
+    ) public virtual onlyRegistryCoordinator {
+        require(_totalStakeHistory[quorumNumber].length == 0, "StakeRegistry.initializeQuorum: quorum already exists");
+        _addStrategyParams(quorumNumber, strategyParams);
+        _setMinimumStakeForQuorum(quorumNumber, minimumStake);
+
+        _totalStakeHistory[quorumNumber].push(OperatorStakeUpdate({
+            updateBlockNumber: uint32(block.number),
+            nextUpdateBlockNumber: 0,
+            stake: 0
+        }));
+    }
 
-    /// @notice Adjusts the `minimumStakeFirstQuorum` -- i.e. the node stake (weight) requirement for inclusion in the 1st quorum.
-    function setMinimumStakeForQuorum(uint8 quorumNumber, uint96 minimumStake) external onlyServiceManagerOwner {
+    function setMinimumStakeForQuorum(
+        uint8 quorumNumber, 
+        uint96 minimumStake
+    ) public virtual onlyServiceManagerOwner quorumExists(quorumNumber) {
         _setMinimumStakeForQuorum(quorumNumber, minimumStake);
     }
 
+    /** 
+     * @notice Adds strategies and weights to the quorum
+     * @dev Checks to make sure that the *same* strategy cannot be added multiple times (checks against both against existing and new strategies).
+     * @dev This function has no check to make sure that the strategies for a single quorum have the same underlying asset. This is a concious choice,
+     * since a middleware may want, e.g., a stablecoin quorum that accepts USDC, USDT, DAI, etc. as underlying assets and trades them as "equivalent".
```

Comment:
I realize these probably weren't part of this PR (it probably just moves the comments?) but:
typo: 'concious' => 'conscious'
and I think 'trades' should be 'treats'